ansible_runner_service.py

```diff
@@ -141,8 +146,32 @@ def setup_common_environment():
     setup_localhost_ssh()
 
 
-def main(test_mode=False):
+def cleanup(scheduler, check_period):
```

Comment:
So my example had some generic names, but I think here we should propagate the ones that come from the config. How about `cleanup` --> `remove_artifacts` and `check_period` --> `frequency.`